numpy/core/src/umath/ufunc_type_resolution.c

```diff
@@ -1342,7 +1327,7 @@ PyUFunc_DefaultLegacyInnerLoopSelector(PyUFuncObject *ufunc,
     PyObject *errmsg;
     int i, j;
 
-    ufunc_name = ufunc->name ? ufunc->name : "(unknown)";
```

Comment:
This changed to now be `<unnamed ufunc>`, but error messages aren't public API anyway.